tee-worker/litentry/primitives/src/identity.rs

```diff
@@ -84,6 +85,18 @@ impl SubstrateNetwork {
 			Self::Litentry => 31,
 			Self::Litmus => 131,
 			Self::Khala => 30,
+			Self::TestNet => 13,
+		}
+	}
+
+	pub fn get_network(prefix: u16) -> Self {
+		match prefix {
+			0 => Self::Polkadot,
+			2 => Self::Kusama,
+			31 => Self::Litentry,
+			131 => Self::Litmus,
```

Comment:
We need to use `LitentryRococo` for this one: we are using rococo for our tests and initial launch.

Whether we will have the protocol on Litmus is unsettled.

But I also understand it might be unpleasant to have the same prefix for Litmus and rococo => I'll check if we can modify this in the SS58Registry.

Reply:
Yeah. Let's keep a note for this point. 'Cause I also found out the value I got when debugging is 131 => Litmus. But I debug with rococo.